code/game/objects/structures/vehicles/clowncart.dm

```diff
@@ -46,6 +61,11 @@
 	..()
 	if(max_health > 100)
 		user << "<span class='info'>It is reinforced with [(max_health-100)/20] bananium sheets.</span>"
+	switch(mode)
+		if(MODE_DRAWING)
+			user << "Currently in drawing mode."
```

Comment:
A message like this should normally be preceded by "[src] is", because it makes no sense out of context.